### DISCUSSION TITLE
Napper iptv sources

### DISCUSSION THREAD

**Discussion by YesTrustMe** [iptv-org/community#315] [Q&A]
This service seems to be promoted heavily by many users in different social media sites, but limited info about them or the service is provided. Has anyone here used them, are they reliable?

**Comment by PopeyeTheSai10r** [comment]
Be aware of services that are heavily advertised. You should see if they offer trials and start with that.

**Comment by BellezaEmporium** [comment]
A "heavily promoted" IPTV source is, most of the time, a bad sign. Usually it's fake accounts that could promote it so that they get more money from people, because you shouldn't really promote your iptv service (even more if it's some illegal sourcing like xtream-codes or something else)

**Comment by YesTrustMe** [comment]
I should clarify. Napper is promoted heavily amongst users in forums, but they don't advertise.  The users seem more like shills .

Napper has a trial, it's a paid trial and support is terrible, but many people swear by them. Of course, I remain skeptical of these people. I was hoping others here might have some experience(s).

**Comment by ifsaputra** [comment]
Yes
